[2025-04-18T18:31:57.593+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-18T18:31:57.651+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_docs_generate manual__2025-04-18T18:27:08.156311+00:00 [queued]>
[2025-04-18T18:31:57.679+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_docs_generate manual__2025-04-18T18:27:08.156311+00:00 [queued]>
[2025-04-18T18:31:57.682+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-04-18T18:31:57.728+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): dbt_tasks.dbt_docs_generate> on 2025-04-18 18:27:08.156311+00:00
[2025-04-18T18:31:57.743+0000] {standard_task_runner.py:72} INFO - Started process 2516 to run task
[2025-04-18T18:31:57.755+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'weather_api_to_snowflake', 'dbt_tasks.dbt_docs_generate', 'manual__2025-04-18T18:27:08.156311+00:00', '--job-id', '278', '--raw', '--subdir', 'DAGS_FOLDER/dbt_snowflake_pipeline.py', '--cfg-path', '/tmp/tmp_ufzocyv']
[2025-04-18T18:31:57.766+0000] {standard_task_runner.py:105} INFO - Job 278: Subtask dbt_tasks.dbt_docs_generate
[2025-04-18T18:31:57.880+0000] {task_command.py:467} INFO - Running <TaskInstance: weather_api_to_snowflake.dbt_tasks.dbt_docs_generate manual__2025-04-18T18:27:08.156311+00:00 [running]> on host b322df8cb9fb
[2025-04-18T18:31:58.213+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_to_snowflake' AIRFLOW_CTX_TASK_ID='dbt_tasks.dbt_docs_generate' AIRFLOW_CTX_EXECUTION_DATE='2025-04-18T18:27:08.156311+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-18T18:27:08.156311+00:00'
[2025-04-18T18:31:58.217+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-04-18T18:31:58.258+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-04-18T18:31:58.260+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt docs generate --project-dir /opt/dbt --profiles-dir /opt/dbt']
[2025-04-18T18:31:58.286+0000] {subprocess.py:99} INFO - Output:
[2025-04-18T18:32:03.908+0000] {subprocess.py:106} INFO - [0m18:32:03  Running with dbt=1.9.4
[2025-04-18T18:32:06.875+0000] {subprocess.py:106} INFO - [0m18:32:06  Registered adapter: snowflake=1.9.2
[2025-04-18T18:32:07.981+0000] {subprocess.py:106} INFO - [0m18:32:07  Found 2 models, 4 data tests, 1 source, 474 macros
[2025-04-18T18:32:07.989+0000] {subprocess.py:106} INFO - [0m18:32:07
[2025-04-18T18:32:07.991+0000] {subprocess.py:106} INFO - [0m18:32:07  Concurrency: 1 threads (target='dev')
[2025-04-18T18:32:07.994+0000] {subprocess.py:106} INFO - [0m18:32:07
[2025-04-18T18:32:09.744+0000] {subprocess.py:106} INFO - [0m18:32:09  Building catalog
[2025-04-18T18:32:14.210+0000] {subprocess.py:106} INFO - [0m18:32:14  Catalog written to /opt/dbt/target/catalog.json
[2025-04-18T18:32:16.724+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-04-18T18:32:16.842+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-04-18T18:32:16.847+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=weather_api_to_snowflake, task_id=dbt_tasks.dbt_docs_generate, run_id=manual__2025-04-18T18:27:08.156311+00:00, execution_date=20250418T182708, start_date=20250418T183157, end_date=20250418T183216
[2025-04-18T18:32:16.933+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-18T18:32:16.994+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
